cura/PrinterOutputDevice.py

```diff
@@ -34,6 +34,12 @@ class ConnectionState(IntEnum):
     busy = 3
     error = 4
 
+class ConnectionType(IntEnum):
+    none = 0
+    usbConnection = 1
+    networkConnection = 2
+    clusterConnection = 3
```

Comment:
WHy also make a difference between a cluster connection? Shouldn't this be more generic (now this is just re-implementing all the output devices that we have).
I think it's better if you have the following types: None, Direct, Local, Cloud

Reply:
I think the names `Direct` and `Local` are not so clear when I first read it. `USB` and `Network` sounds more clear. The distinction between `network` (legacy) and `cluster` (new) can indeed be confusing, but I wonder if there is a good reason to separate them (because of some differences in implementation details maybe?)

Reply:
Well, we don't show a different icon as far as I know. The main difference is mostly due to there being a different number of machines, but that doesn't change the connection. 
So in that regard, the cluster connection type should be removed (Because the connection isn't different, its the amount of machines per output type that's different)

Reply:
There will be a different icon for local networking vs cloud (see designs).